source/configure/plugins-configuration-settings.rst

```diff
@@ -406,6 +406,38 @@ The URL to a running `rtcd <https://github.com/mattermost/rtcd>`__ service insta
 
 This is an optional field. Changing this setting requires a plugin restart to take effect.
 
+Enable call recordings (beta)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: ../_static/badges/ent-selfhosted-only.rst
+  :start-after: :nosearch:
+
+**True**: Allow call hosts to record meeting video and audio on the cloud. Recordings include the entire call window view along with participants' audio track and any shared screen video.
```

Comment:
> Allow call hosts to record meeting video and audio on the cloud.

Maybe not a big deal but I could see a few customers wondering what we mean by "cloud" here. I'd like to stress that recordings are not sent anywhere other than the Mattermost instance where the plugin runs.

Reply:
Very good point, thank you!